face/src/components/chart/BaseChartView.js

```diff
@@ -43,7 +53,7 @@ export default class BaseChartView extends View {
     getTooltipContent() {}
 
     onItemOver(d) {
-        this.showTooltip(d, d3.event.target);
+        this.showTooltip(d, event.target);
```

Comment:
this is wrong `event` object. Even though it works, it may break.

See [docs](https://github.com/d3/d3-selection/blob/master/README.md#event)
> Also beware of conflicts with the window.event global.